### PR TITLE
Issue5613

### PR DESCRIPTION
**Pull request by davidgri**
### PR checklist

- [x ] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [ x] Ran the shell/batch script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates)
- [ x] Filed the PR against the correct branch: master for non-breaking changes and `2.3.0` branch for breaking (non-backward compatible) changes.

### Description of the PR
Added a if-statement to prevent Content-Types on GET-Methods for java

### CODE REVIEW

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/swagger-api/swagger-codegen/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/swagger-api/swagger-codegen/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

**Comment by wing328** [comment]
Please also replace tab with 4-space. Otherwise Travis CI will fail.

**Comment by davidgri** [comment]
The tabs are removed but the build failed because of a connection reset. Somehow npm cant download packages.

**Comment by wing328** [comment]
@davidgri I've restarted the CI job. Let's see how it goes.

**Comment by wing328** [comment]
If you add your fh-dortmund.de email address as your secondary email address in Github, it should fix the issue with the commit authorship.

**Comment by davidgri** [comment]
I have added the email

File: modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractJavaCodegen.java [hunk 1]
```diff
@@ -848,11 +848,13 @@ public void preprocessSwagger(Swagger swagger) {
                         hasFormParameters = true;
                     }
                 }
-                String defaultContentType = hasFormParameters ? "application/x-www-form-urlencoded" : "application/json";
-                String contentType = operation.getConsumes() == null || operation.getConsumes().isEmpty()
-                        ? defaultContentType : operation.getConsumes().get(0);
+              //only add content-Type if its no a GET-Method
```
Comment (wing328) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractJavaCodegen.java]: This line is missing a space and I'll add it later.

**Comment by wing328** [comment]
To close #5613

**Pull request merged**